templates/05-after-static-ip.conf.j2

```diff
@@ -0,0 +1,3 @@
+[Unit]
+Wants={{static_ip_service}}
```

Comment:
Where is this variable `static_ip_service` defined?

Reply:
This should be defined in defaults.yml as part of this role.

Reply:
static_ip_service will be set to fix-eth0.service in group_vars. It can't be set as role default because that wouldn't work in our dev environments that run on DHCP (so don't run fix-eth0.service).